_includes/head-csp.html

```diff
@@ -0,0 +1,24 @@
+<meta http-equiv="Content-Security-Policy" content="
+  default-src https://dpm.demdex.net https://route-default-test-mscherer-matamo.apps.ospo-osci.z3b1.p1.openshiftapps.com/ {{ site.search.host }}; 
+  script-src 'self' 'unsafe-inline' 'unsafe-eval'
+        {{ search_script }} 
+        https://assets.adobedtm.com
```

Comment:
do we really need to remove shas for *everything* and not just the adobe assets?

Reply:
Unfortunately, I've tested it by adding the sha I knew was associated with the light/dark mode and it broke the adobe collection.